code/datums/bodypart_overlays/mutant_bodypart_overlay.dm

```diff
@@ -0,0 +1,116 @@
+///Variant of bodypart_overlay meant to work synchronously with external organs. Gets imprinted upon Insert in on_species_gain
```

Comment:
Were these files straight up missing?

Reply:
Yes, they were in the missed mirror about the bodypart overlays system, created by Time-Green.